invenio_deposit/api.py

```diff
@@ -56,7 +56,11 @@
 
 
 def index(method=None, delete=False):
-    """Update index."""
+    """Decorator to update index.
+
+    :param method: Function wrapped. (Default: None).
```

Comment:
```
. (Default: ``None``)
. (Default: ``'draft'``)
```

1. After dot then I would not add another one after brackets.
2. Inside param description (before last dot).